chain-api/src/types/DexV3Pool.ts

```diff
@@ -356,12 +366,13 @@ export class Pool extends ChainObject {
         this.bitmap,
         state.tick,
         this.tickSpacing,
-        zeroForOne
+        zeroForOne,
+        state.sqrtPrice
       );
 
-      //cap the tick in valid range i.e. -887272 < tick < 887272
-      if (step.tickNext < -887272) throw new Error("Pool lacks enough liquidity to perform this operation");
-      if (step.tickNext > 887272) throw new Error("Pool lacks enough liquidity to perform this operation");
+      //cap the tick in valid range i.e. MIN_TICK < tick < MAX_TICK
+      if (step.tickNext < Pool.MIN_TICK || step.tickNext > Pool.MAX_TICK)
+        throw new Error("Not enough liquidity available in pool");
```

Comment:
```suggestion
      if (step.tickNext < Pool.MIN_TICK || step.tickNext > Pool.MAX_TICK) {
        throw new Error("Not enough liquidity available in pool");
      }
```

For an if-statement this long, prefer braces over one-line syntax. The existing code looks like one-line syntax spread across two lines, which could lead to confusion or mis-interpretation in the future.

Reply:
We have updated the code to throw more specific errors instead of a generic one.